src/SDKs/DataBox/Management.DataBox/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd databox/resource-manager %*
```

Comment:
@kushalvms we moving to .ps1 files form .cmd files so that we can easily add more commandline switches.
Can you model your ps1 from one of the existing one
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1

Reply:
added generate.ps1